plugins/woocommerce/includes/class-wc-order.php

```diff
@@ -2161,27 +2161,45 @@ public function get_total_refunded() {
 	/**
 	 * Get the total tax refunded.
 	 *
-	 * @param bool $include_tax_amount          (optional) Whether to include tax amount in the total. Default true.
-	 * @param bool $include_shipping_tax_amount (optional) Whether to include shipping tax amount in the total. Default true.
-	 *
 	 * @since  2.3
 	 * @return float
 	 */
-	public function get_total_tax_refunded( $include_tax_amount = true, $include_shipping_tax_amount = true ) {
+	public function get_total_tax_refunded() {
 		$cache_key   = WC_Cache_Helper::get_cache_prefix( 'orders' ) . 'total_tax_refunded' . $this->get_id();
 		$cached_data = wp_cache_get( $cache_key, $this->cache_group );
 
 		if ( false !== $cached_data ) {
 			return $cached_data;
 		}
 
-		$total_refunded = $this->data_store->get_total_tax_refunded( $this, $include_tax_amount, $include_shipping_tax_amount );
+		$total_refunded = $this->data_store->get_total_tax_refunded( $this );
 
 		wp_cache_set( $cache_key, $total_refunded, $this->cache_group );
 
 		return $total_refunded;
 	}
 
+	/**
+	 * Get the total shipping tax refunded.
+	 *
+	 * @since  9.8.0
+	 * @return float
+	 */
+	public function get_total_shipping_tax_refunded() {
+		$cache_key   = WC_Cache_Helper::get_cache_prefix( 'orders' ) . 'total_shipping_tax_refunded' . $this->get_id();
+		$cached_data = wp_cache_get( $cache_key, $this->cache_group );
+
+		if ( false !== $cached_data ) {
+			return $cached_data;
+		}
+
+		$total_shipping_tax_refunded = $this->data_store->get_total_shipping_tax_refunded( $this );
```

Comment:
As mentioned I'd add the fallback to 0 here if the function doesn't exist in the data store.